src/flake8_tidy_imports/checks.rs

```diff
@@ -22,3 +26,71 @@ pub fn banned_relative_import(
         None
     }
 }
+
+pub fn name_is_banned<T>(
+    located: &Located<T>,
+    name: String,
```

Comment:
Nit: can this take `&str`?

Reply:
Changing it to take `&str` would result in a needless allocation since the string is firstly formatted in order to be passed to the function and then later has to be owned in order to be part of the CheckKind.

We could change it to instead take two `&str` and and move the `format!` into the function but I don't really see the point.

Reply:
I think the benefit would be that you could then pass in `module: &str` and `name: Alias`, unless I'm misreading, rather than having to extract part of the alias (`let name = &loc_name.node.name`) and pass that in + format it beforehand.

Reply:
Ah yes that makes sense; changed.